src/Shop/ExternalCalculatorService/JsonAdapter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace DDDHH\Shop\ExternalCalculatorService;
+
+use DDDHH\Hexagon\Adapter;
+
+class JsonAdapter extends Adapter
```

Comment:
This is even more shaky. Without a proper `AdapterResponseInterface` an adapter could return anything which in turn makes the purpose of an adapter needless. In most cases an adapter is a domain specific part in some kind of anticorruption layer since it handles the translations between different domain concepts. Hence it should be specific to it's consumer.